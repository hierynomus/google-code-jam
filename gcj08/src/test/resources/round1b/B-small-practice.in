9
365 952 5
330 990 502
10 20 5
765 993 4
591 1000 428
1 1000 2
464 1000 4
10 20 3
61 994 8
