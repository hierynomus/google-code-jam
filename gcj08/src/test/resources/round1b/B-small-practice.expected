Case #1: 129
Case #2: 661
Case #3: 9
Case #4: 82
Case #5: 410
Case #6: 75
Case #7: 131
Case #8: 7
Case #9: 250